meta-balena-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher

```diff
@@ -552,10 +562,8 @@ if [ "$LUKS" = "1" ]; then
     done
 fi
 
-
-resin-device-progress --percentage 100 --state "Post-Provisioning" || true
-
 inform "Shutting down ..."
 shutdown -h now
+report_progress 100 "Post-Provisioning"
```

Comment:
Are you sure you want to move this after the shutdown?

Reply:
Indeed, that's probably caused by a rebase error...